developer-hub/src/polkadot_sdk/mod.rs

```diff
@@ -0,0 +1,126 @@
+//! [![GitHub Repo](https://img.shields.io/badge/github-substrate-2324CC85)](https://github.com/paritytech/polkadot-sdk/blob/master/substrate)
+//!
+//! [`substrate`] is the base blockchain framework used to power the Polkadot SDK. It is a full
+//! toolkit to create sovereign blockchains, including but not limited to those who connect to
+//! Polkadot as parachains.
+//!
+//! #### FRAME
+//!
+//! [![Substrate-license](https://img.shields.io/badge/License-Apache2.0-blue)](https://github.com/paritytech/polkadot-sdk/blob/master/substrate/LICENSE-APACHE2)
+//! [![GitHub Repo](https://img.shields.io/badge/github-frame-2324CC85)](https://github.com/paritytech/polkadot-sdk/blob/master/substrate/frame)
+//!
+//! [`frame`] is the framework used to create Substrate-based runtimes. Learn more about the
+//! distinction of a runtime and client in [`crate::reference_docs::wasm_meta_protocol`].
+//!
+//! #### Cumulus
+//!
+//! [![Cumulus-license](https://img.shields.io/badge/License-GPL3-blue)](https://github.com/paritytech/polkadot-sdk/blob/master/cumulus/LICENSE)
+//! [![GitHub Repo](https://img.shields.io/badge/github-cumulus-white)](https://github.com/paritytech/polkadot-sdk/blob/master/cumulus)
+//!
+//! [`cumulus`] transforms FRAME-based runtimes into Polkadot-compatible parachain runtimes, and
+//! substrate-based client into Polkadot-compatible collators.
+//!
+//! #### Polkadot
+//!
+//! [![Polkadot-license](https://img.shields.io/badge/License-GPL3-blue)](https://github.com/paritytech/polkadot-sdk/blob/master/polkadot/LICENSE)
+//! [![GitHub Repo](https://img.shields.io/badge/github-polkadot-e6007a?logo=polkadot)](https://github.com/paritytech/polkadot-sdk/blob/master/polkadot)
+//!
+//! [`polkadot`] is an implementation of a Polkadot client in Rust, by `@paritytech`. The Polkadot
+//! runtimes are located under the [`fellowship/runtime`](https://github.com/polkadot-fellows/runtimes) repository.
+//!
+//! > [`polkadot`] contains useful links to further learn about Polkadot, **but is in general not
+//! > part of the SDK**, as it is rarely used by developers who wish to build on top of Polkadot.
+//!
+//! #### XCM
+//!
+//! [![XCM-license](https://img.shields.io/badge/License-GPL3-blue)](https://github.com/paritytech/polkadot-sdk/blob/master/polkadot/LICENSE)
+//! [![GitHub Repo](https://img.shields.io/badge/github-XCM-e6007a?logo=polkadot)](https://github.com/paritytech/polkadot-sdk/blob/master/polkadot/xcm)
+//!
+//! [`xcm`], short for "cross consensus message", is the primary format that is used for
+//! communication between parachains, but is intended to be extensible to other use cases as well.
+//!
+//! ### Summary
+//!
+//! The following diagram summarizes how components of the Polkadot-SDK work together:
+#![doc = simple_mermaid::mermaid!("../../../docs/mermaid/polkadot_sdk.mmd")]
+//!
+//! 1. A Substrate-based chain is a blockchain composed of a "Runtime" and a "Client". As noted
```

Comment:
I agree that "Client" sounds counterintuitive, but yeah it's broadly used

Reply:
I 100% agree with "let's settle this now and stick to it" mentality, but prefer client to node. 

Your opinion is fair, but I think it predominantly comes from the PoV of someone in the sdk-node team who's coding this stuff. Truth is, 99% of the substrate/polkadot devs out there never see `struct Client` that you are worried about. 

Contrary, I think between `Client`, `Node` and `Host`, client is the best word as it resembles "Blockchain Client". Moreover, historically, we have used the phrase "Client 
Runtime division" quite a lot. At this point I think doubling down on client is our best bet for not driving more people crazy 🤪 I do mention "Node" in the glossary though. We also have all the crates that compose the subject of this conversation called `sc-*` which stands for client. 

Tagging some people here for opinion. Put 👍 for Client and 👎 for Node please. 

@bkchr @jsdw @joepetrowski @gavofyork @liamaharon @ggwpez

Reply:
I'm calling it Node.

Reply:
My 2c on this is that conceptually that node makes more sense.

In most blockchain systems (incl BTC and ETH) node is the commonly understood term to describe to the software end-users run to handle p2p networking, block importing/building/validation, etc, which seems like the functionality we want to describe here.

Client on the other hand seems more ambiguous/unclear to me. I think it is better for us to align with the rest of the industry, even if we have referred to it as a client in the past.

---

A bit unrelated to this poll, but regarding how to explain the node/runtime divide, rather than describing them as two independent non-overlapping entities like this

<img width="444" alt="Screenshot 2023-11-05 at 10 52 26" src="https://github.com/paritytech/polkadot-sdk/assets/16665596/a14c6ee7-bd1a-4512-8ffe-49ce0fc4fe73">

I think it's easier to understand the runtime as something inert on its own that is loaded into the node, and critical for the node to perform its tasks. Shawn T in this [this old forum post](https://forum.polkadot.network/t/what-are-some-differences-between-substrate-and-cosmos-sdk/1354/8) used a game console & cartridge analogy:

![4a8c6fa5a8721495ada27a9bb254e32c3a6b37b9](https://github.com/paritytech/polkadot-sdk/assets/16665596/69df43e1-e2da-47d1-a525-3a480fbc2ea1)

Reply:
There are no clients in p2p - everyone is equal.

Reply:
My 2c is that client is a bit of an overloaded term anyway (eg I also think of Subxt as a client), and "node" is the word I think I tend to use because it feels more precise.

Reply:
Node it is then, everything in this crate should be refactored to reflect this. Thanks everyone for participating.

Reply:
@liamaharon I have also incorporated the great game console analogy in `polkadot_sdk::substrate`. Sadly not visual at the moment.